### PR TITLE
[PT] bump version and add the working with timezones section

### PR DESCRIPTION
**Pull request by herminiotorres**

### CODE REVIEW

File: pt/lessons/basics/date-time.md [hunk 1]
```diff
@@ -88,20 +88,41 @@ iex> NaiveDateTime.add(~N[2018-10-01 00:00:14], 30)
 ## DateTime
 
 O segundo, como você pode ter adivinhado a partir do título dessa seção, é `DateTime`.
-Ele não tem as limitações vistas em `NaiveDateTime`: ele tem tanto o tempo e data, e suporta fuso horários.
-Mas note o comentário da documentação oficial sobre fuso horário:
+Não possui as limitações mencionadas no `NaiveDateTime`: possui data e hora e suporta fusos horários.
```
Comment (thiamsantos) [inline pt/lessons/basics/date-time.md]:
```suggestion
Não possui as limitações vistas no `NaiveDateTime`: possui data e hora e suporta fusos horários.
```
I'm not sure on this one. But for me `vistas` seems to read a little bit better than `mencionadas`.

File: pt/lessons/basics/date-time.md [hunk 2]
```diff
@@ -88,20 +88,41 @@ iex> NaiveDateTime.add(~N[2018-10-01 00:00:14], 30)
...
+Mas esteja ciente dos fusos horários. Mas note o comentário da documentação oficial sobre fuso horário:
```
Comment (thiamsantos) [inline pt/lessons/basics/date-time.md]:
```suggestion
Mas esteja ciente dos fusos horários. A documentação oficial fala:
```

This seems to be a better translation for `The official docs state:`

File: pt/lessons/basics/date-time.md [hunk 3]
```diff
@@ -88,20 +88,41 @@ iex> NaiveDateTime.add(~N[2018-10-01 00:00:14], 30)
...
+
+> Muitas funções neste módulo requer um fuso horário do banco de dados. Por padrão, é utilizado o fuso horário do banco de dados que é retornado pela função `Calendar.get_time_zone_database/0`, cujo padrão é `Calendar.UTCOnlyTimeZoneDatabase`, que lida apenas com as datas "Etc/UTC" e retorna `{:error, :utc_only_time_zone_database}` para qualquer outro fuso horário.
```
Comment (thiamsantos) [inline pt/lessons/basics/date-time.md]:
```suggestion
> Muitas funções neste módulo requerem um fuso horário do banco de dados. Por padrão, é utilizado o fuso horário do banco de dados que é retornado pela função `Calendar.get_time_zone_database/0`, cujo padrão é `Calendar.UTCOnlyTimeZoneDatabase`, que lida apenas com as datas "Etc/UTC" e retorna `{:error, :utc_only_time_zone_database}` para qualquer outro fuso horário.
```

File: pt/lessons/basics/date-time.md (hunk 1 above)
Comment (herminiotorres) [reply pt/lessons/basics/date-time.md]: @thiamsantos IMHO `vistas` or `mencionadas` it kind the same for me

**Comment by thiamsantos** [comment]
Thanks @herminiotorres :metal:

**Pull request merged**